### ISSUE TITLE
[MAS3.2.1][Keyboard Navigation-Sign in with your azure account] When the user press the enter key on “Sign in with your azure account” focus is not the dialog controls 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1741]
**Actual Result:**
When the user press the enter key on “Sign in with your azure account” focus is not the dialog controls after press tab key focus is moves to the somewhere i.e design guidelines link.

**Expected Result:**
When the user press the enter key on“Sign in with your azure account” focus should be lands on the dialog control only after press the tab key focus should moves to the next interactive control i.e “Don’t have an azure account? sign up.”

**Note:** Same issue exits throughout the application  for all dialog controls 
Same issue repro for below path
Navigate to the Open bot ->Select the Open bot control->Open bot dialog
Welcome Bot Framework Emulator ->Create new bot configuration
Navigate to the Open bot ->Select the Open bot control->Open bot dialog ->Connect the bot 
Bot Explorer->Services->+symbol
Bot Explorer->Services->+symbol->Add language understanding

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Welcome tab screen
3.Navigate to the sign in with your azure account and select it 
4.Dialog gest opened and try to navigate dialog using tab key 
5.Observe the issue

**User Impact:**
Users who rely on keyboard navigation will be affected to navigate the control if keyboard focus is not lands on the expected control keyboard users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}](url)
[MAS3.2.1_Sign in  with your account.zip](https://github.com/microsoft/BotFramework-Emulator/files/3515089/MAS3.2.1_Sign.in.with.your.account.zip)

**Comment by corinagum** [comment]
https://github.com/microsoft/BotFramework-Emulator/tree/corinagum/1741

I have a fix for this and all dialogs that are not maintaining focus within its scope. It looks like the original focus trap code broke in a React bump.

I'm waiting to do a PR for this since @tonyanziano also has dialog changes that I'm waiting for before merging this. I am also hoping e2e tests will get in soon so I can use e2e tests here. :)

**Comment by tonyanziano** [comment]
This will hopefully knock out #1724 as well

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue partiall fixed .Hence we are reactivating this bug 
When we cancel the dialogs focus is moves to bot explorer control instead of respective dialog controls.

**Comment by corinagum** [comment]
Blocked on finishing this one up until #1877 is in.

**Comment by corinagum** [comment]
working on this now.

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version-4.5.2-82590  Issue got fixed hence we are closing this bug.
